api/AudioParam.json

```diff
@@ -162,9 +162,9 @@
                 "version_added": "7.0"
               },
               {
-                "alternative_name": "cancelValuesAndHoldAtTime()",
                 "version_added": true,
-                "version_removed": "6.0"
+                "version_removed": "6.0",
+                "alternative_name": "cancelValuesAndHoldAtTime()"
```

Comment:
I saw the parens here and sent https://github.com/mdn/browser-compat-data/pull/6631. Sorry that this will conflict if landed first, but if you revert the changes to api/AudioParam.json it'll work it.

Reply:
Fixed